stdlib/2and3/audioop.pyi

```diff
@@ -5,38 +5,38 @@ RatecvState = Tuple[int, Tuple[Tuple[int, int], ...]]
 
 class error(Exception): ...
 
-def add(fragment1: bytes, fragment2: bytes, width: int) -> bytes: ...
-def adpcm2lin(adpcmfragment: bytes, width: int, state: Optional[AdpcmState]) -> Tuple[bytes, AdpcmState]: ...
-def alaw2lin(fragment: bytes, width: int) -> bytes: ...
-def avg(fragment: bytes, width: int) -> int: ...
-def avgpp(fragment: bytes, width: int) -> int: ...
-def bias(fragment: bytes, width: int, bias: int) -> bytes: ...
-def byteswap(fragment: bytes, width: int) -> bytes: ...
-def cross(fragment: bytes, width: int) -> int: ...
-def findfactor(fragment: bytes, reference: bytes) -> float: ...
-def findfit(fragment: bytes, reference: bytes) -> Tuple[int, float]: ...
-def findmax(fragment: bytes, length: int) -> int: ...
-def getsample(fragment: bytes, width: int, index: int) -> int: ...
-def lin2adpcm(fragment: bytes, width: int, state: Optional[AdpcmState]) -> Tuple[bytes, AdpcmState]: ...
-def lin2alaw(fragment: bytes, width: int) -> bytes: ...
-def lin2lin(fragment: bytes, width: int, newwidth: int) -> bytes: ...
-def lin2ulaw(fragment: bytes, width: int) -> bytes: ...
-def max(fragment: bytes, width: int) -> int: ...
-def maxpp(fragment: bytes, width: int) -> int: ...
-def minmax(fragment: bytes, width: int) -> Tuple[int, int]: ...
-def mul(fragment: bytes, width: int, factor: float) -> bytes: ...
+def add(__fragment1: bytes, __fragment2: bytes, __width: int) -> bytes: ...
+def adpcm2lin(__fragment: bytes, __width: int, __state: Optional[AdpcmState]) -> Tuple[bytes, AdpcmState]: ...
+def alaw2lin(__fragment: bytes, __width: int) -> bytes: ...
+def avg(__fragment: bytes, __width: int) -> int: ...
+def avgpp(__fragment: bytes, __width: int) -> int: ...
+def bias(__fragment: bytes, __width: int, __bias: int) -> bytes: ...
+def byteswap(__fragment: bytes, __width: int) -> bytes: ...
+def cross(__fragment: bytes, __width: int) -> int: ...
+def findfactor(__fragment: bytes, __reference: bytes) -> float: ...
+def findfit(__fragment: bytes, __reference: bytes) -> Tuple[int, float]: ...
+def findmax(__fragment: bytes, __length: int) -> int: ...
+def getsample(__fragment: bytes, __width: int, __index: int) -> int: ...
+def lin2adpcm(__fragment: bytes, __width: int, __state: Optional[AdpcmState]) -> Tuple[bytes, AdpcmState]: ...
+def lin2alaw(__fragment: bytes, __width: int) -> bytes: ...
+def lin2lin(__fragment: bytes, __width: int, __newwidth: int) -> bytes: ...
+def lin2ulaw(__fragment: bytes, __width: int) -> bytes: ...
+def max(__fragment: bytes, __width: int) -> int: ...
+def maxpp(__fragment: bytes, __width: int) -> int: ...
+def minmax(__fragment: bytes, __width: int) -> Tuple[int, int]: ...
+def mul(__fragment: bytes, __width: int, __factor: float) -> bytes: ...
 def ratecv(
-    fragment: bytes,
-    width: int,
-    nchannels: int,
-    inrate: int,
-    outrate: int,
-    state: Optional[RatecvState],
-    weightA: int = ...,
-    weightB: int = ...,
+    __fragment: bytes,
```

Comment:
From the CPython source, it looks like that these were all intentionally kept as positional-only when this module was converted to Argument Clinic—kind of weird since with so many arguments keyword arguments are obviously useful. There's no justification in https://bugs.python.org/issue20133